### PR TITLE
WE-225: Announce Decommission of Messaging Tools

### PR DESCRIPTION
**Pull request by bkemper**
Announced decommission as a banner on each tool.

Screen Shot 2021-07-19 at 6 50 48 PM

### CODE REVIEW

File: src/components/ShutdownAnnouncement.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './ShutdownAnnouncement.scss';
+
+const ShutdownAnnouncement = ({ tool }) => (
+  <div className="ShutdownAnnouncement">
+    <p className="ShutdownAnnouncement__headline">Announcement</p>
```
Comment (nicklemmon) [inline src/components/ShutdownAnnouncement.js:6]: This should be an `h3` I think

File: src/components/ShutdownAnnouncement.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './ShutdownAnnouncement.scss';
+
+const ShutdownAnnouncement = ({ tool }) => (
+  <div className="ShutdownAnnouncement">
+    <p className="ShutdownAnnouncement__headline">Announcement</p> 
+    <p className="ShutdownAnnouncement__text">
+      We will be shutting down the {tool} at the end of July 2021. 
+      If you need help or have any questions, please{" "}
+      <a className="ShutdownAnnouncement__link" href="//app.sparkpost.com/dashboard?supportTicket=true">
```
Comment (nicklemmon) [inline src/components/ShutdownAnnouncement.js:10]: Should this open in a new tab?

**Review by nicklemmon — CHANGES_REQUESTED** [review]
@bkemper I had two tiny points of feedback. The heading thing is less important than the new tab thing I think.

**Comment by bkemper** [comment]
@nicklemmon Thanks for the feedback.  Ready for another 👀 .

**Review by nicklemmon — APPROVED** [review]
@bkemper ✅

**Pull request merged**